### PR TITLE
created an AUTHORS file.

### PR DESCRIPTION
**Pull request by ThakeeNathees**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,30 +39,6 @@ It's a [WebAssembly](https://webassembly.org/) build of the VM compiled using [e
 Note that in the webassembly version of the language, some features (input, relative import, etc.) have disabled, has
 limited memory allocations, and the IO calls might be slower.
 
-### On Your System
```
Comment (ThakeeNathees) [inline README.md:42]: I'm moving this tutorial to [documentations](https://thakeenathees.github.io/pocketlang/), this page (README.md) is an overviwe of the language, and also if someone knows how to build it from source, they already know how to run the script. 

However thanks for the contribution, For the lack of tutorials, consider adding some to the [docs/pages](https://github.com/ThakeeNathees/pocketlang/tree/master/docs/).

Comment (xSavitar) [reply README.md:42]: Okay @ThakeeNathees. Thanks!

**Comment by xSavitar** [comment]
> CC @xSavitar

Can you use the email <alangiderick@gmail.com> instead? Thanks!

**Review by xSavitar — CHANGES_REQUESTED** [review]
Thanks for adding me :)

Just a quick change of email request.

**Pull request merged**